src/sentry/search/events/filter.py

```diff
@@ -613,7 +613,12 @@ def convert_search_filter_to_snuba_query(
     elif name in ARRAY_FIELDS and search_filter.value.is_wildcard():
         # Escape and convert meta characters for LIKE expressions.
         raw_value = search_filter.value.raw_value
-        like_value = raw_value.replace("%", "\\%").replace("_", "\\_").replace("*", "%")
+        like_value = (
+            raw_value.replace("\\", "\\\\")
```

Comment:
Could we include a comment here explaining why this needs to be done? eg.
```suggestion
            # Slashes need to be double escaped so they're interpreted as a literal slash
            raw_value.replace("\\", "\\\\")
```